components/signersForm.tsx

```diff
@@ -94,7 +94,7 @@ const SignersForm: FC<{
     let ops: ownersForm[] = [];
 
     if (
-      typeof effectivePeriod !== "undefined" &&
+      !!effectivePeriod &&
       Number(effectivePeriod) != props.contract.effective_period.toNumber()
```

Comment:
there is a bug here when go to settings page.
`KT1TjeAtpMbZ8gTdWmFF984CYYUnJg5nD9Xb`
```
Unhandled Runtime Error
TypeError: props.contract.effective_period.toNumber is not a function

Source
components/signersForm.tsx (98:65) @ toNumber

   96 | if (
   97 |   !!effectivePeriod &&
>  98 |   Number(effectivePeriod) != props.contract.effective_period.toNumber()
      |                                                             ^
   99 | ) {
  100 |   ops.push({ adjustEffectivePeriod: Number(effectivePeriod) });
  101 | }
Call Stack
getOps
components/signersForm.tsx (298:10)
```

Reply:
it fixed!